src/sql/parts/notebook/models/notebookModel.ts

```diff
@@ -86,7 +87,12 @@ export class NotebookModel extends Disposable implements INotebookModel {
 		if (this.notebookOptions.layoutChanged) {
 			this.notebookOptions.layoutChanged(() => this._layoutChanged.fire());
 		}
-		this._defaultKernel = notebookOptions.defaultKernel;
+		if (this._providerId && (this._providerId.toLowerCase() === SQL_NOTEBOOK_PROVIDER.toLocaleLowerCase())) {
+			this._defaultKernel = this._sqlKernel;
```

Comment:
How about sqlKernel is not enabled by default

Reply:
👍

Reply:
Is it specific to both when python is not installed and sqlKernel feature flag is not enabled ?

Reply:
We should test both scenarios